### PR TITLE
Updating docs on embedding and images with the documentation

### PR DESCRIPTION
**Pull request by Iskandarmah**
Adding documentation on embedding GIANT dashboards and images needed in the documentation.

### CODE REVIEW

**Comment by Iskandarmah** [comment]
@smileEng

File: giant/embedding-giant-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+- Advanced Embed
+
+- Chart Embed
+
+## Basic Embed
+
+If you want the audience to be able to view the data exactly as what you see in the dashboard, use **Basic embed**.
+
+First, click the three lines shown at the top right side of the GIANT dashboard page.
```
Comment (tchyong) [inline giant/embedding-giant-dashboard.md]: three lines??
this is a hamburger icon.
If this name is not familiar, you can call it as more option/action

**Comment by tchyong** [comment]
@smileEng the screen shot use show ```peter_8excite```. Got any concern on disclose that?

File: giant/embedding-giant-dashboard.md (hunk 1 above)
Comment (smileEng) [reply giant/embedding-giant-dashboard.md]: hamburger icon is fine

**Comment by smileEng** [comment]
@Iskandarmah Please do resolve the conflicts

**Comment by smileEng** [comment]
> @smileEng the screen shot use show `peter_8excite`. Got any concern on disclose that?

**Pull request closed without merging**